Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
         }
     }
 
+    func outputName(at index: Int) -> String {
+        precondition(index < outputCount,
+            "Output index out of bounds when getting outputName.")
+        var ssaName = ""
+        if let id = self.id {
+            ssaName = "%\(id)"
+        } else {
+            ssaName = "%\(ObjectIdentifier(self))"
+        }
+        if outputCount > 1 {
+            ssaName += ".\(index)"
+        }
+        return ssaName
+    }
+
+    var outputName: String {
+        switch outputCount {
+        case 0: return ""
+        case 1: return "\(outputName(at: 0))"
+        default:
+            do {
```

Comment:
Why use a `do` block here?

Reply:
ah, habit from C++ to put `case` blocks within {}. Removed it.